deploy/kubedirector/kubedirector.hpe.com_kubedirectorclusters_crd.yaml

```diff
@@ -601,9 +601,8 @@ spec:
                         type: string
                       statefulSet:
                         type: string
-                      upgradingMembers:
-                        type: object
-                        nullable: true
+                      membersUpgrading:
```

Comment:
Seems like we need to call this something slightly different since it is used both for upgrades and rollbacks. Hmm. I mean the obvious name is `membersUpgradingOrRollingBack` but that is a liiiiitle too long.

Let's leave it as it is for now (unless you have another great name in mind), because I'm still not sure we want to have this field in the status. I'll have another comment about that elsewhere.

Reply:
`upgradingMembersCount` ?